app-misc/blink1/blink1-1.98-r1.ebuild

```diff
@@ -76,23 +78,29 @@ src_install() {
 	if use doc; then
 		dodoc docs/README.md
 	fi
+
 	if use tool; then
 		if use doc; then
 			dodoc docs/{blink1-tool.md,blink1-tool-tips.md,blink1-hid-commands.md,blink1-mk2-tricks.md}
 		fi
+
 		if use examples; then
-			docinto /usr/share/doc/"${PF}"/examples
+			docinto examples
```

Comment:
not your issue, check line 101 though

Reply:
b2e49fa8bdb4d37d863db972680539dc579128a9